src/Endpoints/Shipments.php

```diff
@@ -70,7 +70,11 @@ protected function getShipmentsResource(string $apiMethod, string $message = '')
             $apiMethod
         );
 
-        $shipment = collect($response->data->shipments)->first();
+        $shipment = collect($response->data->shipments)
+            ->map(function ($shipment) {
+                return json_decode(json_encode($shipment), true);
+            })
+            ->first();
 
```

Comment:
What is the reason for this change? This shouldn't be necessary.

Reply:
Reverted